src/main/scala/com/lucidworks/spark/example/ml/SVMExampleScala.scala

```diff
@@ -0,0 +1,161 @@
+
+    val stdTokLowerSchema = "{ \"analyzers\": [{ \"name\": \"std_tok_lower\", \"tokenizer\": { \"type\": \"standard\" },\n" +
+      "                \"filters\": [{ \"type\": \"lowercase\" }]}],\n" +
+      "  \"fields\": [{ \"regex\": \".+\", \"analyzer\": \"std_tok_lower\" }]}\n"
+
+    val numFeatures = cli.getOptionValue("numFeatures", DEFAULT_NUM_FEATURES).toInt
+    val numIterations = cli.getOptionValue("numIterations", DEFAULT_NUM_ITERATIONS).toInt
+
+    def RowtoLab(row: Row, numFeatures: Int, inputCols: Array[String], stdTokLowerSchema: String ): LabeledPoint = {
+      var textAnalyzer: LuceneTextAnalyzer = new LuceneTextAnalyzer(stdTokLowerSchema)
+      var hashingTF = new HashingTF(numFeatures)
+      var normalizer = new Normalizer()
+      val polarity = row.getString(row.fieldIndex("polarity"))
+      var fields = new java.util.HashMap[String, String]()
+      for(i <- 0 until inputCols.length){
+        val value = row.getString(row.fieldIndex(inputCols(i)))
+        if (value != null) {
+          fields.put(inputCols(i), value)
+        }
+      }
+      val analyzedFields = textAnalyzer.analyzeJava(fields)
+      var terms = new java.util.LinkedList[String]()
+      analyzedFields.values().asScala.toList.foreach(v => terms.addAll(v))
+      val sentimentLabel = if (("0" == polarity)) 0.toDouble else 1.toDouble
+      new LabeledPoint(sentimentLabel, normalizer.transform(hashingTF.transform(terms)))
+    }
+
+    var trainingData = trainingDataFromSolr.rdd.map(row => RowtoLab(row, numFeatures, inputCols, stdTokLowerSchema))
+    var standardScaler = new StandardScaler().fit(trainingData.map(x => x.features))
+    var trainRDD = trainingData.map(x => new LabeledPoint(x.label, standardScaler.transform(x.features)))
+    trainRDD = trainRDD.persist(StorageLevel.MEMORY_ONLY_SER)
+    val model = SVMWithSGD.train(trainRDD, numIterations)
+
+    options = immutable.HashMap("zkhost" -> cli.getOptionValue("zkHost", DefaultZkHost),
+      "collection" -> cli.getOptionValue("collection", DefaultCollection),
+      "query" -> "+test_polarity:[* TO *] +tweet_txt:[* TO *]",
+      "fields" -> "id,test_polarity,tweet_txt")
+
+    var testDataFromSolr = sqlContext.read.format("solr").options(options).load()
+    testDataFromSolr = testDataFromSolr.withColumnRenamed("test_polarity", "polarity")
+    testDataFromSolr.show
+    val testVectors = testDataFromSolr.rdd.map(row => RowtoLab(row, numFeatures, inputCols, stdTokLowerSchema)).map(x => new LabeledPoint(x.label, standardScaler.transform(x.features)))
+
+    val scoreAndLabels =  testVectors.map(p => {
+      val score = model.predict(p.features)
+      System.out.println(">> model predicted: " + score + ", actual: " + p.label)
+      new (Double, Double)(score, p.label)
+    })
+
+    val metrics = new BinaryClassificationMetrics(scoreAndLabels)
+    val auROC = metrics.areaUnderROC
+    System.out.println("Area under ROC = " + auROC)
```

Comment:
Use scala member println for this http://www.scala-lang.org/api/rc/index.html#scala.Console$